webapp/home/subdomains/proteomics/content/dda_standardised_tools.py

```diff
@@ -53,6 +51,22 @@
             }
         ],
         "button_link": "https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/msstats/msstats/",
+    },
+    {
+        "title_html": '<code>LFQ Analyst</code>',
+        "description_html": '<p>Analyze and Visualize Label-Free Proteomics output from MaxQuant.</p>',
+        "inputs": [
+            {
+                'datatypes': ['txt'],
+                'label': 'Protein groups (MaxQuant output)',
+            },
+            {
+                'datatypes': ['txt'],
+                'label': 'Experimental Design Matrix (MaxQuant output)',
+            }
+        ],
+        "button_link": "https://usegalaxy.org.au/root?tool_id=interactive_tool_lfqanalyst_2",
+        "view_link": "Shah AD, Goode RJA, Huang C, Powell DR, Schittenhelm RB. LFQ-Analyst: An easy-to-use interactive web-platform to analyze and visualize proteomics data preprocessed with MaxQuant. https://doi.org/10.1021/acs.jproteome.9b00496",
```

Comment:
This should be a URL, not a reference. e.g. https://doi.org/10.1021/acs.jproteome.9b00496.
Also, when the user clicks on the "View" (eye) icon for a tool, I don't think they would expect to read the article?